src/db/DB_Upgrade.h

```diff
@@ -201,6 +201,12 @@ const std::vector<wxString> dbUpgradeQuery =
         PRAGMA user_version = 10;
     )",
 
+    // Upgrade to version 11
```

Comment:
Please update database submodule to version taggeg "v11" then regenerate DB_* files